acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/privacy/multitenancy/MultiTenancyValidationFailAcceptanceTest.java

```diff
@@ -0,0 +1,189 @@
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.junit.WireMockRule;
+import org.apache.tuweni.bytes.Bytes;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+public class MultiTenancyValidationFailAcceptanceTest extends AcceptanceTestBase {
+  private BesuNode node;
+  private final ObjectMapper mapper = new ObjectMapper();
+  private Cluster multiTenancyCluster;
+
+  private static final int ENCLAVE_PORT = 1080;
+  private static final String PRIVACY_GROUP_ID = "Z3JvdXBJZA==";
+  private static final String ENCLAVE_KEY = "B1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=";
+  private static final String OTHER_ENCLAVE_KEY = "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=";
+  private final Address senderAddress =
+      Address.wrap(Bytes.fromHexString(accounts.getPrimaryBenefactor().getAddress()));
+
+  @Rule public WireMockRule wireMockRule = new WireMockRule(options().port(ENCLAVE_PORT));
+
+  @Before
+  public void setUp() throws Exception {
+    final ClusterConfiguration clusterConfiguration =
+        new ClusterConfigurationBuilder().awaitPeerDiscovery(false).build();
+    multiTenancyCluster = new Cluster(clusterConfiguration, net);
+    node =
+        besu.createNodeWithMultiTenantedPrivacy(
+            "node1",
+            "http://127.0.0.1:" + ENCLAVE_PORT,
+            "authentication/auth_priv.toml",
+            "authentication/auth_priv_key");
+    multiTenancyCluster.start(node);
+
+    final String token =
+        node.execute(permissioningTransactions.createSuccessfulLogin("failUser", "pegasys"));
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+  }
+
+  @After
+  public void tearDown() {
+    multiTenancyCluster.close();
+  }
+
+  @Test
+  public void sendTransactionShouldFailWhenPrivateFromNotMatchEnclaveKey()
+      throws JsonProcessingException {
+    final PrivateTransaction validSignedPrivateTransaction =
+        getValidSignedPrivateTransaction(senderAddress, OTHER_ENCLAVE_KEY);
+    retrievePrivacyGroupEnclaveStub();
+    final Transaction transaction =
+        privacyTransactions.sendRawTransaction(
+            getRLPOutput(validSignedPrivateTransaction).encoded().toHexString());
+    node.verify(priv.multiTenancyValidationFail(transaction, JsonRpcError.ENCLAVE_ERROR));
+  }
+
+  @Test
+  public void sendTransactionShouldFailWhenPrivacyGroupDoesNotContainEnclaveKey()
+      throws JsonProcessingException {
+    final PrivateTransaction validSignedPrivateTransaction =
+        getValidSignedPrivateTransaction(senderAddress, ENCLAVE_KEY);
+    retrievePrivacyGroupEnclaveStub();
+    final Transaction transaction =
+        privacyTransactions.sendRawTransaction(
+            getRLPOutput(validSignedPrivateTransaction).encoded().toHexString());
+    node.verify(priv.multiTenancyValidationFail(transaction, JsonRpcError.ENCLAVE_ERROR));
+  }
+
```

Comment:
Also need tests for privDistributeRawTransaction

Reply:
Is that required?  I'm just testing the fail paths through MultiTenancyPrivacyController.

Reply:
Yeah I think so as we can't test the MultiTenancyPrivacyController directly. It's possible that privDistributeRawTransaction could change at some point to just call into the enclave and bypass the validation